std/math/emulated/field_assert.go

```diff
@@ -111,22 +111,24 @@ func (f *Field[T]) AssertLimbsEquality(a, b *Element[T]) {
 	}
 }
 
-// EnforceWidth enforces that the bitlength of the value is exactly the
-// bitlength of the modulus. Any newly initialized variable should be
-// constrained to ensure correct operations.
-func (f *Field[T]) EnforceWidth(a *Element[T]) {
+// enforceWidth enforces the width of the limbs. When modWidth is true, then the
+// limbs are asserted to be the width of the modulus (highest limb may be less
```

Comment:
I am a bit confused by the comments; 

"highest limb may be less than full limb width" --> may or must? 

Also, it's unclear; does this method guarantee that the constrain emulated element `e` is mod reduced? (`e < q`) ?

Reply:
May be. So usually the maximum width of the limb is given by the NbBits parameter (64 bits etc.). However, it usually doesn't divide emulated modulus evenly. Then the highest limb has fewer than NbBits bits. But if divides evenly, then is exactly NbBits.

This method doesn't ensure that is mod reduced. It only ensures (if argument `modWidth=true`) that the width of the element is same as the modulus. So in general it could either be `x` or `x-p`, but for the arithmetic either case is sufficient.

For ensuring that value is less than modulus, there is the `AssertIsLessEqualThan` method.